cave/plot/parallel_coordinates.py

```diff
@@ -71,17 +101,21 @@ def plot_n_configs(self, num_configs, params):
         # Get n most run configs
         if num_configs == -1:
             num_configs = len(all_configs)
-        configs_to_plot = sorted(all_configs, key=lambda x:
-        len(self.original_rh.get_runs_for_config(x)),
-                                 reverse=True)[:num_configs]
         self.logger.debug("Plotting %d configs.", num_configs)
-        self.validated_rh = self.validator.validate_epm(configs_to_plot,
+        self.validated_rh = self.validator.validate_epm(all_configs,
                                                         'train+test', 1,
                                                         runhistory=self.original_rh)
+        configs_to_plot = sorted(all_configs, key=lambda x: self.validated_rh.get_cost(x))
+        # What about scenarios where quality is the value to optimize? shouldn't min and max be switched then?
```

Comment:
@shukon If we talk about SMAC then yes. However if someone would want to use cave with data not obtained through SMAC, it might be worth having a flag that tells the analysis part if maximization or minimization is the goal.